config/locales/en.yml

```diff
@@ -12,7 +12,7 @@ en:
           default: true
         queue:
           description: Which ActiveJob queue to put events on
-          default: ":default"
+          default: ":dfe_analytics"
```

Comment:
This is now at odds with https://github.com/DFE-Digital/dfe-analytics/blob/86ebf54c11da3473a632bf299ed0418ef24b27f1/lib/dfe/analytics.rb#L79